INSTALL

```diff
@@ -22,5 +23,6 @@ get .pem key, use it to ssh into instance
 upload backend folder and requirements.txt, run.sh, startup.sh into instance
 ./startup.sh
 ./run.sh
+add custom TCP 5000 in security group

```

Comment:
the handout says that we should specify what kind of OS we're using. I'm not sure how that works in EC2 but is there a way we can see what kind of server/vm the backend will be running on?